app/routes/scenarios--gen-results.js

```diff
@@ -119,13 +123,29 @@ function spawnAnalysisProcess (projId, scId, opId) {
     '-e', `STORAGE_SECRET_KEY=${config.storage.secretKey}`,
     '-e', `STORAGE_BUCKET=${config.storage.bucket}`,
     '-e', `STORAGE_REGION=${config.storage.region}`,
-    '-e', 'CONVERSION_DIR=/conversion',
-    config.analysisProcess.container
+    '-e', 'CONVERSION_DIR=/conversion'
   ];
 
+  let service = config.analysisProcess.service;
+  switch (service) {
+    case 'docker':
+      break;
+    case 'hyper':
+      baseOptions.push = (
+        '-e', `HYPER_ACCESS=${config.analysisProcess.hyperAccess}`,
+        '-e', `HYPER_SECRET=${config.analysisProcess.hyperSecret}`
+      );
+      break;
+    default:
+      console.log(`${service} is not a valid option. The analysis should be run on 'docker' or 'hyper'. Check your config file or env variables.`);
+  }
+
+  // Append the name of the image last
+  let args = baseOptions.concat(config.analysisProcess.container);
```

Comment:
Maybe use always the same approach when adding elements to the array. Call it `args` from the start and always use `push`, seems more consistent.